dali/test/python/test_utils_tensorflow.py

```diff
@@ -184,33 +184,51 @@ def run_dataset_in_graph(dali_datasets, iterations):
 
     with tf.compat.v1.Session() as sess:
         sess.run(initializers)
-        for _ in range(iterations):
-            dataset_results.append(sess.run(ops_to_run))
+        try:
```

Comment:
This try/except could be a helper function, and be reused.

Reply:
Careful what you wish for :P

Reply:
Added a context manager for that, don't know a better way (I just saw similar example in pip's source today).